conda/cli/conda_argparse.py

```diff
@@ -199,6 +226,45 @@ def _check_value(self, action, value):
         else:
             super()._check_value(action, value)
 
+    @staticmethod
+    def _get_attempted_override_error(name: str, summary: str):
+        return (
+            f"The plugin '{name}: {summary}' is trying to override "
+            f"the built-in command {name}, which is not allowed. Please uninstall this plugin "
+            "to stop seeing this error message"
+        )
+
+    def _get_subcommand_plugin(self) -> Namespace | None:
+        """
+        Return a Namespace object containing the ``CondaSubcommand`` object if the current
+        subcommand belongs to a registered plugin
+        """
+        if len(sys.argv) > 1:
+            name = sys.argv[1]
+            for subcommand in self._subcommands:
+                if subcommand.name == name:
+                    if name.lower() in BUILTIN_COMMANDS:
+                        error_msg = self._get_attempted_override_error(
+                            subcommand.name, subcommand.summary
+                        )
+                        log.error(error_msg)
+                    else:
+                        return Namespace(plugin_subcommand=subcommand)
```

Comment:
I don't think there's any value in defining this as a helper method, lets's inline it below

Reply:
In my opinion, the value comes from being able to more easily read the `parse_args` method. It also helps to decrease the overall code complexity of that method.

But, it seems like Jannis also thinks this is a good idea, so I will inline it. We can always refactor later 😉.